provisioning_templates/provision/autoyast_default.erb

```diff
@@ -22,21 +22,54 @@ oses:
   <networking>
     <dns>
       <hostname><%= @host.name %></hostname>
+      <% if @host.subnet.respond_to?(:dhcp_boot_mode?) -%>
+      <dhcp_hostname config:type="boolean">false</dhcp_hostname>
+      <dhcp_resolv config:type="boolean">false</dhcp_resolv>
+      <domain><%= @host.domain %></domain>
+      <nameservers config:type="list">
+        <nameserver><%= @host.subnet.dns_primary %></nameserver>
+        <nameserver><%= @host.subnet.dns_secondary %></nameserver>
```

Comment:
I think we should check for presence of those nameservers, at least the secondary one - check the other templates: something like this:
<% if @host.subnet.dns_secondary.present? -%>
        <nameserver><%= @host.subnet.dns_secondary %></nameserver>
<% end -%>